index.html

```diff
@@ -1350,6 +1348,115 @@ <h3>alsoKnownAs</h3>
+              <code>id</code> property DID value.
+            </dd>
+            <dd>
+              A resolving party SHOULD retain the DIDs from the <code>id</code> and 
+              <code><a>sameAs</a></code> properties to ensure any subsequent 
+              interactions with any of the DIDs they contain are correctly handled 
+              as logically equivalent DIDs (e.g. retain all variants in a database so an 
+              interaction with any one maps to the same underlying account).
+            </dd>
+          </dl>
+  
+          <div class="note" title="Equivalence and sameAs">
+            <p>
+              <code><a>sameAs</a></code> was chosen as the name for this property because 
+              from a graph modeling standpoint the intended semantics are identical to 
+              the <code>owl:sameAs</code> property. <code><a>sameAs</a></code> is a much 
+              stronger form of equivalence than <code><a>alsoKnownAs</a></code> because the 
+              equivalence MUST be guaranteed by the governing DID method. <code><a>sameAs</a></code> 
+              represents a full graph merge because the same DID document describes both 
+              the <code><a>sameAs</a></code> DID and the <code>id</code> property DID.
+            </p>
+          </div>
+  
+        </section>
+
+      <section>
+        <h3>canonicalId</h3>
+          <p>
+            The <code><a>canonicalID</a></code> property is identical to the 
+            <code><a>sameAs</a></code> property except: a) it accepts only a single 
+            DID as its value, and b) that DID is defined to be the canonical DID 
+            for the DID subject within the scope of the containing DID document. 
+          </p>
+  
+          <dl>
+            <dt><dfn>canonicalId</dfn></dt>
+            <dd>
+              The value of <code><a>canonicalID</a></code> MUST be a DID conforming to the 
+              requirements in Section 3.1.
+            </dd>
+            <dd>
+              The relationship is a statement that the <code><a>canonicalID</a></code> DID 
+              value is logically equivalent to the <code>id</code> property DID value and 
+              that the <code><a>canonicalID</a></code> DID value is defined by the 
+              <a>DID Method</a> to be the canonical DID for the DID subject in the scope 
+              of the containing DID document.
+            </dd>
+            <dd>
+              A <code><a>canonicalID</a></code> DID value MUST be produced by the same 
+              <a>DID Method</a> as the <code>id</code> property DID value.
+            </dd>
+            <dd>
+              A conforming <a>DID Method</a> specification MUST guarantee that the 
+              <code><a>canonicalID</a></code> DID value is logically equivalent to the 
+              <code>id</code> property DID value.
+            </dd>
+            <dd>
+              A resolving party SHOULD use the <code><a>canonicalID</a></code> DID value 
```

Comment:
again. not sure this is testable... I think it could probably be weakened further to a note / recommendation, or exempted from the test suite and that would be a way to resolve this.